tools/clang/unittests/SPIRV/CMakeLists.txt

```diff
@@ -25,10 +25,13 @@ target_link_libraries(clang-spirv-tests
   clangCodeGen
   clangFrontend
   clangSPIRV
+  dxcompiler
   effcee
   SPIRV-Tools
   )
 
+add_dependencies(clang-spirv-tests dxcompiler)
```

Comment:
Is this for resolving the linking-in-twice problem? Worth some comments here.

Reply:
Yes. Done.